modules/system/console/WinterUtil.php

```diff
@@ -292,6 +292,63 @@ protected function utilPurgeThumbs()
         }
     }
 
+    protected function utilPurgeResizedCache()
+    {
+        if (!$this->confirmToProceed('This will PERMANENTLY DELETE all images in the resized directory.')) {
+            return;
+        }
+
+        $totalCount = 0;
+        $uploadsPath = Config::get('filesystems.disks.local.root', storage_path('app'));
+        $uploadsPath .= '/resized';
```

Comment:
Are you able to get this to support remote disks by looking at the `cms.storage.resized` configuration: https://github.com/wintercms/winter/blob/develop/config/cms.php#L329-L333?